src/js/components/PlacementConstraintsPartial.js

```diff
@@ -155,8 +138,24 @@ export default class PlacementSection extends Component {
               <option value="">Select</option>
               {this.getOperatorTypes()}
             </FieldSelect>
+            <FieldHelp>Specify where your app will run.</FieldHelp>
             <FieldError>{operatorError}</FieldError>
           </FormGroup>
+          <FormGroup
+            className={commonFieldsClassNames}
+            required={true}
+            showError={Boolean(fieldNameError)}
+          >
+            {fieldLabel}
+            <FieldInput
+              name={`constraints.${index}.fieldName`}
+              type="text"
+              placeholder="hostname"
+              value={constraint.fieldName}
+            />
+            <FieldHelp>Ex: zone, region, hostname</FieldHelp>
```

Comment:
Could we extract this as a PR? I feel like we're blocking a committed feature because of this being snuck in.